core/integration/testdata/modules/go/ifaces/main.go

```diff
@@ -13,6 +13,11 @@ func (m *Caller) Test(
 	// +optional
 	run string,
 ) error {
+	_, err := dag.CurrentModule().Source().Export(ctx, ".")
```

Comment:
Interesting, but makes sense, seems like this friction could help nudge folks towards more explicit inputs, which is a good thing

Reply:
Oh this is actually just kind of silly and not something most users would need to do. It's just because this test is the special one that invokes `go test` in itself so it needs its `go.mod`, `dagger.gen.go`, etc. in its workdir (before this the test started failing because it couldn't find a `go.mod`).

Obviously the long-term solution for this is official `dagger test` support, but in the meantime the choice was to do this mildly silly thing or to re-write all those interface tests to either shell out to the dagger CLI or use raw gql queries. Neither of which felt worth it right now when this escape hatch was sitting there 🙂